### DISCUSSION TITLE
Pressing escape key while transitioning to results (based on stable expectations) should go to results

### DISCUSSION THREAD

**Discussion by RadekMojzis** [ppy/osu#15706] [Ideas] [answered]
Right now a lot of people in standard instantly press escape after finishing the map to instantly jump to score screen immediately. 

If you do this in laser it takes you straigth to song select (so the game forgets things like UR and hit distribution) which is quite frustrating because of my muscle memory from stable

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/15726
